node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/StoredBoolean.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.node.admin.task.util.file;
+
+import com.yahoo.vespa.hosted.node.admin.component.TaskContext;
+
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+/**
+ * Class wrapping a boolean stored on disk.
+ *
+ * <p>The implementation is compatible with {@link StoredInteger} when absence or 0 means false.
+ *
+ * @author hakonhall
+ */
+public class StoredBoolean {
+    private static Logger logger = Logger.getLogger(StoredBoolean.class.getName());
+
+    private final UnixPath path;
+
+    /** The parent directory must exist. Value is false by default. */
+    public StoredBoolean(Path path) {
+        this.path = new UnixPath(path);
+    }
+
+    public boolean value() {
+        return path.readUtf8FileIfExists().map(String::trim).map(s -> !"0".equals(s)).orElse(false);
+    }
+
+    /** Sets value to true. */
+    public void set(TaskContext context) {
```

Comment:
Add a method that can sets it equal to the given `boolean` instead/also?

Reply:
ok, also